lisa/features/disk_ephemeral.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.feature import Feature
+
+FEATURE_NAME_DISK = "DiskEphemeral"
+
+
+class DiskEphemeral(Feature):
+    @classmethod
+    def name(cls) -> str:
+        return FEATURE_NAME_DISK
+
+    @classmethod
+    def can_disable(cls) -> bool:
+        return True
+
+    def _switch(self, enable: bool) -> None:
+        raise NotImplementedError()
+
+    def disable(self) -> None:
+        self._switch(False)
```

Comment:
It's worth to have a SwitchableMixin class to put those methods together. `disable`, `enable`, `_switch`.

Reply:
But the names are confusing with `enabled` from base feature class. Is there any better idea?

Reply:
`disable`, `enable`, `_switch` should be applicable to every feature, right? If so, we can just move them to Feature class. If not, we should identify what set for features could potentially use it and create a mixin specific to this purpose.

Reply:
There are two levels of disable/enable. One is used before deploying, another one is used after deployed and in the node. I will think about to support the deploying level variables. For now, let's have a separated mixin to support the second level disable/enable.